src/App.vue

```diff
@@ -61,7 +61,11 @@
 
     @XOn(['SearchRequestChanged'])
     setWysiwygContext(payload: InternalSearchRequest | null): void {
-      window.wysiwyg?.setContext({ query: payload?.query });
+      const { wysiwyg } = window;
+      if (wysiwyg) {
+        const query = this.$x.spellcheckedQuery! || payload?.query;
```

Comment:
The `||` operator must be used instead of `??` as when no spellcheck is applied the result of `$x.spellcheckedQuery` is an empty string instead of undefined; the `!` is required to avoid linting issues.